docs/agent/secrets.md

```diff
@@ -24,6 +24,11 @@ in your configuration (not the key), in any section (`init_config`, `instances`,
 
 Secrets are supported in every configuration backend: file, etcd, consul ...
 
+Secrets are also supported in `datadog.yaml`. The agent will first load the
```

Comment:
nit: I find it a bit weird that we mention this offhand in the `Defining secrets in check configurations` section. What do you think of adding a new section for this instead? :)

i.e. something like:

```
## Defining secrets in configurations
### Integration configurations
[...]
### Main Agent configuration (`datadog.yaml`)
[your new paragraph, that just says that it's basically the same thing for datadog.yaml]
```

Reply:
I'll update the title of the section. I don't want to create two sections since what would be in the `check` section would also apply to datadog.yaml. And I would like to highlight that it's the same behavior.